examples/NAT.cs

```diff
@@ -1,379 +0,0 @@
-/*
```

Comment:
Maybe it's worth keeping the old implementation, renaming it to something like "Simple NAT" perhaps, since it has the nice quality of being succinct, and serves as an example that one can get useful behaviour using relatively little code. (This also serves as the [example in the front README](https://github.com/niksu/pax#what-does-a-pax-processor-look-like).)